hardware/smart_utils.py

```diff
@@ -14,15 +14,11 @@
 import subprocess
 import sys
 
-import six
-
 from hardware import smart_utils_info
 
 
 def _parse_line(line):
-    line = line.strip()
-    if isinstance(line, six.binary_type):
-        line = line.decode(errors='ignore')
+    line = line.strip().decode(errors='ignore')
     return line
```

Comment:
Why not "return line.strip().decode(errors='ignore')" ?

Reply:
it would just save 1 line and complicate the code visually
to be honest, I'm not a fan of complex or long 'return', just to save few lines of code